### DISCUSSION TITLE
Parameters are not getting mapped using ApplysimpleTemplate method

### DISCUSSION THREAD

**Discussion by umesh14398** [idea-statica/ideastatica-public#388] [Help Q&A] [answered]
Hi all,

I am trying to apply a template to a new connection using the **ApplySimpleTemplate** method.  I could be able to apply the template but parameters are not appearing in the developer tab. I could see the operations in the design tab but not the parameters that exist in the template file. Please find the images after applying a template. Is it not possible to have parameters in the developer tab if the template is applied using ApplySimpleTemplate method.?
<img width="820" alt="image" src="https://github.com/idea-statica/ideastatica-public/assets/43907961/0617a526-72a4-43b1-8013-8a21c15c7d93">

<img width="953" alt="image" src="https://github.com/idea-statica/ideastatica-public/assets/43907961/c19bdbb7-3e7b-400f-8fc2-a2fbfec73494">

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @umesh14398, yes unfortunately this is not currently working. We disabled this because we do not currently have a way of managing multiple 'simple' templates that have identical parameters. I will add this as an issue and we will see if we can enable it.
